tests/dummy/app/app.js

```diff
@@ -1,13 +1,14 @@
 import Ember from 'ember'
-import Resolver from 'ember/resolver'
+const {Application} = Ember
 import loadInitializers from 'ember/load-initializers'
+import Resolver from 'ember/resolver'
```

Comment:
We need to update this line to: `import Resolver from './resolver'` it was missed in previous upgrades.